### DISCUSSION TITLE
Humans spawned through debug console have faulty radio

### DISCUSSION THREAD

**Discussion by TheNinjaScout** [FakeFishGames/Barotrauma#14519] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When you spawn a human via the debug console, they automatically join your crew with a random job assigned, but their radio doesn't allow them to hear orders or general radio text messages from anybody in the original crew at a long range. The problem resides in the headsets themselves.

### Reproduction steps

1. Start a multiplayer match
2. Spawn in some humans through the debug console
3. Attempt to give orders to the new crew members
4. Notice how they can only hear you when you're nearby and viceversa despite having a radio
5. Swap your headset with theirs
6. Notice how you can no longer give orders to anyone in the original crew at long range but can now be heard by any of the newly spawned-in humans when using radio / giving orders

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
I think it was addressed https://discord.com/channels/103209684680323072/668819422919524352/1277594887767720016

**Reply by TheNinjaScout** [reply]
That seems to be exactly it, good to know!

**Comment by TheNinjaScout** [comment]
It appears that the issue is already fixed internally, so I'm closing this
